server/entity/direction.go

```diff
@@ -27,6 +28,22 @@ func Facing(e world.Entity) cube.Direction {
 	return 0
 }
 
+// Bearing splits the 360 degree to sections and returns the section number by yaw.
+func Bearing(yaw float64, max int64) int64 {
```

Comment:
This isn't used anywhere.

Reply:
> This isn't used anywhere.

This was for getting the rotation of decoration. But Sofe suggested me to use another approach.

Reply:
Function removed.